libvast/vast/command.hpp

```diff
@@ -42,6 +42,9 @@ class command {
   /// Maps names of config parameters to their value.
   using option_map = std::map<std::string, caf::config_value>;
 
+  /// Iterates over CLI arguments.
+  using const_iterator = std::vector<std::string>::const_iterator;
```

Comment:
Perhaps `argument_iterator` is a more meaningful name for the iterator type. Then you can also use more idiomatic variable names (`begin` instead of `args_begin`) because the types have more semantics.

When someone reads `command::const_iterator`, they have to read the documentation to figure out what's being iterated over, whereas `command::argument_iterator` is more self-explanatory.

Reply:
+1 for `argument_iterator`, I didn't think of that.